packages/client-ts/src/client.ts

```diff
@@ -208,6 +210,39 @@ class Client implements IClient {
     return this.config
   }
 
+  public async getPublicProfile(did: string,
+    contextName: string,
+    profileName: string = "basicProfile",
+    fallbackContext: string | null = "Verida: Vault",
+    useCache: boolean = true): Promise<ProfileDocument> {
+    if (this.config.readOnlyDataApiUri) {
+      // Try to fetch the profile from our data API
+      const fetchUri = `${this.config.readOnlyDataApiUri}/${did}/${contextName}/profile_public/${profileName}?ignoreCache=true`
```

Comment:
`ignoreCache=true` is this params specified by the API consumer

Reply:
Should be resolved.